### PR TITLE
add 2fa page missing in institutional sign in

### PR DESCRIPTION
**Pull request by DanielPalafox**
https://trello.com/c/L1WNUVfT/6758-add-new-page-link-shibboleth-signin-for-new-institutional-linking-orcid-angular

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[Add new page link /shibboleth/signin for new institutional linking orcid-angular](https://trello.com/c/L1WNUVfT/6758-add-new-page-link-shibboleth-signin-for-new-institutional-linking-orcid-angular)

File: src/app/two-factor/pages/two-factor/two-factor.component.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          if (res.errors[0].includes('verification')) {
+            this.showBadVerificationCode = true
+          } else {
+            this.showBadRecoveryCode = true
+          }
+        } else if (res.redirectUrl) {
+          this.navigateTo(res.redirectUrl)
+        }
+      },
+      (error) => console.log(error)
```
Comment (leomendoza123) [inline src/app/two-factor/pages/two-factor/two-factor.component.ts:76]: You are already using the ERROR_REPORT.STANDARD_VERBOSE to report this to the user and the console, so no need to handle the error here as well

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -97,6 +97,11 @@ const routes: Routes = [
     loadChildren: () =>
       import('./search/search.module').then((m) => m.SearchModule),
   },
+  {
+    path: ApplicationRoutes.twoFactor,
+    loadChildren: () =>
```
Comment (leomendoza123) [inline src/app/app-routing.module.ts:102]: A guard is required here, right? So no one can load this module without the required context

File: src/app/two-factor/pages/two-factor/two-factor.component.ts (hunk 1 above)
Comment (DanielPalafox) [reply src/app/two-factor/pages/two-factor/two-factor.component.ts:76]: Yes, you are right, great catch. I will remove it

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**